### DISCUSSION TITLE
选择拷贝文本的时候不灵敏，并且选择文本之后无法取消拷贝光标

### DISCUSSION THREAD

**Discussion by Trace132** [community/community#73923] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.137.1 (114555847)
Device: iPhone 13 mini (iOS 16.7.2)
</details>

**Comment by kauaneiras** [comment]
🚀

**Comment by eliperkins** [comment]
Hey @Trace132 , thanks for your feedback! Please forgive me as I'm using a translation service here. It looks like you mentioned that selecting or copying text is difficult, and that "Copy" cannot be dismissed after selecting text. Is that correct?

If so, can you send me a screenshot of where you're seeing this behavior in the iOS app?

Thanks!

**Comment by ettaboyle** [comment]
Hi @Trace132, thanks for participating and providing feedback!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. 

**Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting.** We’ll be locking any posts in languages other than English for now on. Please note that team members are sometimes able to use translation services, but that is not always reliable and not something we can do at scale. 

Please post in english in the future. Thank you!
